src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -206,7 +207,8 @@ private async Task<string> Fetch(Func<HttpRequestMessage> requestFactory, string
                 return value404;
             }
 
-            return await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync();
+            await EnsureSuccessStatusCode(response);
+            return await response.Content.ReadAsStringAsync();
 
             async Task<HttpResponseMessage> SendRequest(HttpRequestMessage request)
             {
```

Comment:
Consider use an `AsyncLocal<string>` to store and retrieve the current request URL.

Reply:
To store the request url, we need reference type(e.g. AsyncLocal<_CustomClass_>), is it acceptable to use a local variable to store request url here? It may request lot's of change to pass AsyncLocal<_CustomClass_> to here.

Reply:
Sure, since now the code does not touch `CredentialHandler.cs`